sdk/src/main/resources/account/api/accountApi.yaml

```diff
@@ -1500,7 +1500,7 @@ paths:
       tags:
         - transaction
       summary:  Returns the paginated list of forging stakes.
-      description: Returns the paginated list of forging stakes.
+      description: Returns the paginated list of forging stakes. It will be disabled after Forger Stake V2 will be activated.
```

Comment:
Maybe change comment to something like "This endpoint is disabled after Forger Stake V2 activation.", so that it is not related to some point in time

Reply:
Done